style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: NexaLight;
+  src: url('./Font/NexaLight.otf');
+}
+
+@font-face {
+  font-family: NexaBlack;
+  src: url('./Font/NexaBlack.otf');
+}
+
+@font-face {
+  font-family: NexaBold;
+  src: url('./Font/NexaBold.otf');
+}
+
+* {
+  margin-top: 10px;
+}
+
+.banner-section {
+  background-color: black;
+  display: flex;
+  margin: auto;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.container-banner {
+  display: flex;
+}
+
+.main-title {
+  display: none;
+  font-size: 25px;
+  line-height: 63px;
+  color: black;
+  margin: 0 0 15px;
+}
+
+.banner-wrapper {
+  display: flex;
+}
+
+.left-section {
+  width: 50%;
+  text-align: left;
+  margin: 50px 0px 40px 40px;
+  flex: 1;
+}
+
+.banner-image {
+  max-width: 400px;
+  width: 100%;
+  align-self: flex-start;
+}
+
+.right-section {
+  width: 50%;
+  margin: 50px 40px 40px 0px;
+  flex: 1;
+}
+
+.title {
+  padding: 0px 6px 0px 0px;
+  font-size: 44px;
+  letter-spacing: normal;
+  font-weight: 700;
+  color: #c6cdd9;
+  line-height: 66px;
+  font-family: NexaBold;
+}
+
+.description {
+  font-family: NexaLight;
+  font-size: 20px;
+  font-weight: 300;
+  color: white;
+  margin: 0 10px 35px 0;
+  line-height: 34px;
+}
+
+.learnmore-link-button {
+  font-family: NexaBold;
+  font-size: 14px;
+  padding: 15px 30px;
+  font-weight: 300;
+  color: #d9e2fd;
+  border-radius: 8px;
+  background-color: blue;
+  border: 1.5px solid blue;
```

Comment:
instead of using color directly we use hex code for that, If we use variable format than it is good @aeshapatel025